Makefile

```diff
@@ -166,7 +166,7 @@ bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 deploy-setup: install
-	@EXCLUSIONS=() hack/deploy-setup.sh $(NAMESPACE)
+	@EXCLUSIONS= hack/deploy-setup.sh $(NAMESPACE)
```

Comment:
I think we can remove `@EXCLUSIONS=` as well.

Reply:
It was pushed originally with `@EXCLUSIONS=()` removed, I re-added `@EXCLUSIONS=` according to @zshi-redhat  comment above, WDYT should it be removed or stay at it is?

Reply:
Sorry, I didn't see zshi's comments. I think it can be just removed. The `EXCLUSIONS` is only for the case when you want to not apply any manifest file, e.g. when you want to run the operator locally for development. And in `deploy-setup` case, we want to deploy all of them.

Reply:
Updated